src/TokensTracker/Trackers/TrackerBase.cs

```diff
@@ -129,7 +129,8 @@ protected TransferRecord GetTransferRecord(UInt160 asset, Array stateItems)
             return stateItems.Count switch
             {
                 3 => new TransferRecord(asset, @from, to, null, amountItem.GetInteger()),
-                4 when (stateItems[3] is ByteString tokenId) => new TransferRecord(asset, @from, to, tokenId, amountItem.GetInteger()),
+                4 when stateItems[3] is ByteString tokenId => new TransferRecord(asset, @from, to, tokenId, amountItem.GetInteger()),
+                4 when stateItems[3] is Integer tokenId => new TransferRecord(asset, from, to, tokenId.GetSpan().ToArray(), amountItem.GetInteger()),
```

Comment:
Why `tokenId` is an `Integer`?

Reply:
Because it can be used in absence of neo-project/neo#1879?

Reply:
neo-project/neo#2810

Reply:
When the developer calls the contract, the tokenid is used as an Integer. The notified field is Integer。

Contract Example：
`
public class Test : SmartContract
{
    [DisplayName("Test")]
    public static event Action<ByteString> OnTest;

    public static void CallEvent(ByteString tokenId)
    {
        OnTest(tokenId);
    }
}
`

`
sb.EmitDynamicCall(contract, "callEvent", 1);
`
type of event :  Integer

`
sb.EmitDynamicCall(contract, "callEvent", ((BigInteger)1).ToByteArray());
`
type of event :  ByteString

Reply:
When the developer calls the contract, the tokenid is used as an Integer. The notified field is Integer。

event Example：
```
public static event Action<ByteString> OnTest;

public static void CallEvent(ByteString tokenId)
{
        OnTest(tokenId);
}
```

`
sb.EmitDynamicCall(contract, "callEvent", 1);
`
type of the field  in event :  Integer


`
sb.EmitDynamicCall(contract, "callEvent", ((BigInteger)1).ToByteArray());
`
type of the field in event:  ByteString

Reply:
> When the developer calls the contract, the tokenid is used as an Integer. The notified field is Integer。
> 
> event Example：
> 
> ```
> public static event Action<ByteString> OnTest;
> 
> public static void CallEvent(ByteString tokenId)
> {
>         OnTest(tokenId);
> }
> ```
> 
> `sb.EmitDynamicCall(contract, "callEvent", 1);` type of the field in event : Integer
> 
> `sb.EmitDynamicCall(contract, "callEvent", ((BigInteger)1).ToByteArray());` type of the field in event: ByteString


For https://github.com/neo-project/neo/pull/2810, we can ignore these "invalid" transactions and close this PR, right?